src/app/features/charts/components/median-barchart/median-barchart.component.spec.ts

```diff
@@ -54,8 +54,8 @@ describe('Component: BarChart - Median', () => {
   let component: MedianBarChartComponent;
   let element: HTMLElement;
 
-  beforeEach(waitForAsync(() => {
-    TestBed.configureTestingModule({
+  beforeEach(waitForAsync(async () => {
+    await TestBed.configureTestingModule({
```

Comment:
Await promise, so there is not a warning from newly enabled `@typescript-eslint/no-floating-promises` rule